jdaviz/core/template_mixin.py

```diff
@@ -2102,7 +2104,8 @@ def _update_subset(self, subset, attribute=None):
                 self.items = self.items + [self._subset_to_dict(subset)]  # noqa
         else:
             # 'type' can be passed manually rather than coming from SubsetUpdateMessage.attribute
-            if attribute in ('style', 'type'):
+            # This will be None if triggered by SubsetCreateMessage
+            if attribute in ('style', 'type') or attribute is None:
```

Comment:
Any reason we can't condense this?  (Not sure if this will complain about `==` instead of `is`).

```suggestion
            if attribute in ('style', 'type', None):
```

Reply:
I think that's fine.